test/test_transform.py

```diff
@@ -670,6 +670,19 @@ def test_add_inames_for_unused_hw_axes(ctx_factory):
             parameters={"n": n})
 
 
+def test_rename_argument_of_domain_params():
+    knl = lp.make_kernel(
+            "{[i, j]: 0<=i<n and 0<=j<m}",
+            """
+            y[i, j] = 2.0f
+            """)
+
+    knl = lp.rename_argument(knl, "n", "N")
+    knl = lp.rename_argument(knl, "m", "M")
+
+    print(lp.generate_code_v2(knl).device_code())
```

Comment:
You could assert that the old name is not part of the code.

Reply:
0b8589a